components/button/button.js

```diff
@@ -16,15 +16,15 @@ function Button({ children, mods = [], size, href, onClick, type, variation, dis
   }
 
   if (size) {
-    mods.push(`size_${size}`);
+    mods.push(`size-${size}`);
```

Comment:
just as an additional example (not in button, but for some future components):
`app-bar_show-icon-left` or `app-bar_show-icon_left`?